readme.md

```diff
@@ -96,6 +96,10 @@ All the below methods accept a value and returns a boolean for whether the value
 
 Keep in mind that [functions are objects too](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions).
 
+##### .numericString(value)
+
+Returns `true` for a string that contains a number.
```

Comment:
That's not entirely correct. It returns true for a string that **represents** a number. If it was *contains*, then this would be valid `is.numericString('foo 4 bar') //=> true`.